src/common/types/chainner-scope.ts

```diff
@@ -29,9 +29,11 @@ struct PyTorchModel {
     scale: int(1..),
     inputChannels: int(1..),
     outputChannels: int(1..),
-    modelType: string,
+    arch: string,
     size: string,
+    subType: string,
 }
+let PyTorchFaceModelArchs = "GFPGAN" | "RestoreFormer";
```

Comment:
I think we should use the namespace feature here.
```suggestion
let PyTorchModel::FaceArchs = "GFPGAN" | "RestoreFormer";
```